### ISSUE TITLE
Ocelot .Net 5.0 On kube message: namespace:etlbigdataservice:etl-dataresource-service Unable to use ,it is invalid

### ISSUE THREAD

**Issue by GeorGeWzw** [ThreeMammals/Ocelot#1447]
Ocelot .Net 5.0 On kube message: namespace:etlbigdataservice:etl-dataresource-service Unable to use ,it is invalid,
The namespace{etlbigdataservice} exists and The Service {etl-dataresource-service}  exists.
![image](https://user-images.githubusercontent.com/46982191/109743998-90757280-7c0c-11eb-8ddc-3f0f935805c1.png)
![image](https://user-images.githubusercontent.com/46982191/109743999-90757280-7c0c-11eb-8903-31c7d1d4d573.png)

warn: Ocelot.Provider.Kubernetes.KubernetesServiceDiscoveryProvider[0]
      requestId: 0HM6TTIS49UFF:00000003, previousRequestId: no previous request id, message: namespace:etlbigdataservice:etl-dataresource-service Unable to use ,it is invalid. Address must contain host only e.g. localhost and port must be greater than 0
warn: Ocelot.Responder.Middleware.ResponderMiddleware[0]
      requestId: 0HM6TTIS49UFF:00000003, previousRequestId: no previous request id, message: Error Code: ServicesAreEmptyError Message: There were no services in NoLoadBalancer errors found in ResponderMiddleware. Setting error response for request path:/doc/ETL.DataResource/v1.0/swagger.json, request method: GET

**Comment by GeorGeWzw** [comment]
![image](https://user-images.githubusercontent.com/46982191/109744391-2c9f7980-7c0d-11eb-9329-d92e8222ecae.png)

**Comment by luisfernandomoraes** [comment]
Guys, any response about this issue?

**Comment by luisfernandomoraes** [comment]
Hey @GeorGeWzw do you have some troubleshooting for this issue?

**Comment by GeorGeWzw** [comment]
@luisfernandomoraes Yes,I have fiexd the probiem , This problem is caused by kubernetes' lack of configuration permission, You can refer to it 
https://ocelot.readthedocs.io/en/latest/features/kubernetes.html
https://kubernetes.io/docs/reference/access-authn-authz/rbac/#permissive -rbac-permissions
I did it in kubernetes
‘kubectl create clusterrolebinding permissive-binding \
--clusterrole=cluster-admin \
--user=admin \
--user=kubelet \
--group=system:serviceaccounts’;
but,this  is not recommended by kubernetes

**Comment by luisfernandomoraes** [comment]
Tks so much @GeorGeWzw! I have forget this step in my production environment.

**Comment by luisfernandomoraes** [comment]
Tks so much @GeorGeWzw! I have forget this step in my environment.

**Comment by raman-m** [comment]
Hi @KawhiWei !
We don't support .NET 5 anymore since [21.0.0](https://github.com/ThreeMammals/Ocelot/releases/tag/21.0.0) release on Nov 21, 2023, and Microsoft .NET team doesn't support .NET 5 too since November 2023.
